src/components/estimated-call/EstimatedCallInfo.tsx

```diff
@@ -0,0 +1,77 @@
+import {Statuses, StyleSheet} from '@atb/theme';
+import {useTranslation} from '@atb/translations';
+import {View} from 'react-native';
+import {formatDestinationDisplay} from '@atb/travel-details-screens/utils';
+import {usePreferences} from '@atb/preferences';
+import {LineChip, LineChipServiceJourney} from './LineChip';
+import {ThemeIcon} from '../theme-icon';
+import {PinInvalid} from '@atb/assets/svg/mono-icons/map';
+import {ThemeText} from '@atb/components/text';
+import {DestinationDisplay} from '@atb/api/types/generated/journey_planner_v3_types';
+
+type EstimatedCallItemDeparture = {
+  destinationDisplay?: DestinationDisplay;
+  cancellation: boolean;
+  predictionInaccurate: boolean;
+  serviceJourney: LineChipServiceJourney;
+};
+type EstimatedCallProps = {
+  departure: EstimatedCallItemDeparture;
+  ignoreSituationsAndCancellations?: boolean;
+  messageType?: Exclude<Statuses, 'valid'>;
+  testID?: string;
+};
```

Comment:
Making "duplicate" types like this isn't a pattern we've used much before, but I think it makes sense here:

It uses the same structure as journey planner for estimated calls, but only requires the fields that the component actually needs. This way, we can pass an estimated call from departure details and departures, even thought they don't use the exact same types.

Reply:
If they are just a subset of a domain model from journey planner, then I would use the Typescript utility type `Pick` to formalise the dependency between them.